docs/csharp/programming-guide/interop/walkthrough-office-programming.md

```diff
@@ -43,7 +43,11 @@ Both languages enable embedding of type information, which allows deployment of
 This walkthrough demonstrates these features in the context of Office programming, but many of these features are also useful in general programming. In the walkthrough, you use an Excel Add-in application to create an Excel workbook. Next, you create a Word document that contains a link to the workbook. Finally, you see how to enable and disable the PIA dependency.  
   
 ## Prerequisites  
+<<<<<<< HEAD
 You must have Microsoft Office Excel and Microsoft Office Word installed on your computer to complete this walkthrough.  
+=======
+ You must have Microsoft Office Excel 2007 or later and Microsoft Office Word 2007 or later installed on your computer to complete this walkthrough.  
+>>>>>>> Updates
   
```

Comment:
@BillWagner I'll take care of that.

Also, @mairaw floated the idea of keeping the breaking changes topic.

> I think we should keep the topic but remove version from the file name and make the topic cumulative over time. I'll redirect all the C# and VB topics from VS.140 (Vs 2015) version to here soon.

Since we removed the 2013 version in the csharp docs and the 2015 version is currently in the VB area (and has VB is in the title), I need guidance on what should be done with the breaking changes topic.